src/MobileBookingHeader/OneWayTrip.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import * as React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+import idx from 'idx';
+
+import type OneWayTrip_booking from './__generated__/OneWayTrip_booking.graphql';
```

Comment:
Aha, this is maybe why:

```diff
-  import type OneWayTrip_booking from './__generated__/OneWayTrip_booking.graphql';
+  import type { OneWayTrip_booking } from './__generated__/OneWayTrip_booking.graphql';
```

🙄

Reply:
yes! I've just tied it, this is the reason.

Reply:
Makes sense. It's imported as `any` type. You should tweak your Flow configuration to warn you about this.

Reply:
Fixed.

@mrtnzlml are you referring to `no-weak-types`?

Reply:
This one maybe: https://flow.org/en/docs/linting/rule-reference/#toc-untyped-type-import

But it's probably too complicated to setup this rule (we are not able to do this yet in our codebase).